.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1105,6 +1108,205 @@ stages:
       condition: succeededOrFailed()
       continueOnError: true
 
+- stage: asan_tests_windows_profiler
+  condition: > 
+    and(
+      succeeded(), 
+      eq(variables['isBenchmarksOnlyBuild'], 'False'),
+      eq(dependencies.generate_variables.outputs['generate_variables_job.generate_variables_step.IsProfilerChanged'], 'True')
+    )
+  dependsOn: [master_commit_id, generate_variables, build_windows_tracer, build_windows_profiler]
+  variables:
+    masterCommitId: $[ stageDependencies.master_commit_id.fetch.outputs['set_sha.master']]
+  jobs:
+  - template: steps/update-github-status-jobs.yml
+    parameters:
+      jobs: [Win]
+
+  - job: Win
+    pool:
+      vmImage: windows-2019
```

Comment:
We should almost certainly be using `windows-2022` now (I'll be surprised if this works on 2019, as .NET 7 SDK isn't supported there). We _might_ want to move these to the azure scale set too, but that can come later if required

Reply:
Well, it works 😅 
Update VMs here (also for Linux) 68145d6ca29d49a9df8643bedf4dc536fd4332fd

We can move to azure scale sets in future, I'll add it to TODO list.